portal/templates/flask_user/register.html

```diff
@@ -123,36 +128,52 @@ <h3 class="modal-title" id="myModalLabel">About Using Google or Facebook for Tru
     })(window.location.search.substr(1).split('&'));
 
     if (qs["email"]) $("#email").val(qs["email"]);
-    $("#password").popover({"content": "<b>Password must:</b><ul id='pwdHintList'><li class='text-info eight-characters'>Have at least 8 characters</li><li class='text-info lowercase-letter'>Have at least one lowercase letter</li><li class='text-info uppercase-letter'>Have at least one uppercase letter</li><li class='text-info one-number'>Have at least one number</li></ul>", "placement": "top", "html": true, "title": "Requirements", "template": '<div class="password-popover popover" role="tooltip"><div class="arrow"></div><h3 class="popover-title"></h3><div class="popover-content"></div></div>', "container": ".password-wrapper", "trigger": "focus"}).click(function(e) {
-        if (!$(".password-popover").is(":visible")) $(this).popover("show");
+
+    $("#password").popover({"content": "<b>Password must have at least:</b><ul id='pwdHintList'><li class='text-info eight-characters'>Eight characters</li><li class='text-info lowercase-letter'>One lowercase letter</li><li class='text-info uppercase-letter'>One uppercase letter</li><li class='text-info one-number'>One number</li></ul>", "placement": "top", "html": true, "title": "Requirements", "template": '<div class="password-popover popover" role="tooltip"><div class="arrow"></div><h3 class="popover-title"></h3><div class="popover-content"></div></div>', "container": ".password-wrapper", "trigger": "manual"}).click(function(e) {
```

Comment:
Could you wrap any visible text with translation tags (ie `{%trans%}`)? We'll have to come up for a strategy for translating content in static javascript files at some point, but I think we'll still need to tag strings like this.